### PR TITLE
Point AAT at Preprod Vault section

### PR DESCRIPTION
**Pull request by Louisblack**
### Change description ###
* It should point to the preprod vault section.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by lgonczar** [comment]
send-letter also points to `test` from `aat`. This change will break it.

**Comment by Louisblack** [comment]
Any chance of merging this in now that AAT and Test won't be peered from 5PM today?

File: infrastructure/aat.tfvars [hunk 1]
```diff
@@ -0,0 +1,2 @@
+testing_support = "true"
```
Comment (alectronic0) [inline infrastructure/aat.tfvars]: this could actually be false now

**Comment by Louisblack** [comment]
Updated PR to only point to preprod and updated title and description to match this. 
We are now using our vault secrets so do not need testing end points open.

**Comment by Louisblack** [comment]
Looks like Jenkins and Git are having a bad time. I’ll have a butchers when I get in. Although when I looked yesterday, the master build was failing on Sonarqube so that might need to be looked at.

**Pull request closed without merging**